vadl/test/resources/snapshots/rv64im/InstrInfoTableGen.td

```diff
@@ -1730,7 +1730,7 @@ let Size = 4;
 let CodeSize = 4;
 
 let OutOperandList = ( outs X:$rd );
-let InOperandList = ( ins X:$rs1, X:$rs2 );
+let InOperandList = ( ins X:$rs2, X:$rs1 );
```

Comment:
fyi @benjaminkasper99 because the operands are determined much earlier the order of the operands has changed. However, the emission is still correct. Just keep that in mind when writing the parser tests.